app/src/main/java/iband/com/ibandplayerdemo/PlayerActivity.java

```diff
@@ -0,0 +1,440 @@
+package iband.com.ibandplayerdemo;
+
+
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+import android.view.WindowManager;
+import android.widget.AdapterView;
+import android.widget.ImageView;
+import android.widget.ListView;
+import android.widget.ProgressBar;
+import android.widget.SeekBar;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.iband.playersdk.IBandBasePlayer;
+import com.iband.playersdk.IBandError;
+import com.iband.playersdk.IBandPlayer;
+import com.iband.playersdk.IBandPlayerVariant;
+import com.iband.playersdk.IBandPlayerView;
+import com.iband.playersdk.IBandSDK;
+import com.iband.playersdk.IBandStream;
+
+import java.util.ArrayList;
+
+import iband.com.ibandplayerdemo.adapter.VariantAdapter;
+import iband.com.ibandplayerdemo.adapter.VariantHolder;
+
+public class PlayerActivity extends AppCompatActivity implements IBandPlayer.Listener, IBandStream.Listener {
+
+    private static final String STREAM_ID = "5994219bbc454f6edf961315";
```

Comment:
Duplicate, you already put stream ID on: app/src/main/AndroidManifest.xml

Reply:
done